core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/JsonSerializer.java

```diff
@@ -51,6 +60,8 @@ JsonObject writeTextAnnotation(TextAnnotation ta) {
         writeString("id", ta.getId(), json);
         writeString("text", ta.getText(), json);
         writeStringArray("tokens", ta.getTokens(), json);
+        if ( doWriteTokenOffsets )
+            writeTokenOffsets( TOKENOFFSETS, ta.getView(ViewNames.TOKENS), json);
```

Comment:
Not necessary: readTextAnnotation computes the token offsets from the raw text. The output is simply a convenience for people who want to use non-CCG code to reconstruct the annotations.